src/main/java/frc/robot/Robot.java

```diff
@@ -116,6 +157,7 @@ public void autonomousInit() {
     if (m_autonomousCommand != null) {
       m_autonomousCommand.start();
     }
+    Robot.m_climber.singleInit();
```

Comment:
There are 3 phases of initialization for the climber:
1) The constructor which runs when the robot starts up
2) This `singleInit` method which gets run when the match starts (I hope that's when autonomousInit is run?)
3) The `climberInit` method which is run every time we switch to climber mode

I wanted singleInit to set up the soft limits because we don't want to reset those every time we flip in and out of climber mode. I didn't want to set them up in the constructor in case someone needed to adjust the legs to their starting position after the bot was already on.